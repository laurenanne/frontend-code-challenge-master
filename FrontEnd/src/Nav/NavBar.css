.navbar {
    background-color:#d15c9e;
    color: rgb(101, 99, 99)

  }
  
  
  .nav-item {
    width: 100px;
    text-align: center;
  }
  
  
  nav a {
    color: white;
    text-decoration: none;
  }
  
  
  nav a.active:not(.navbar-brand) {
    color: white;
    font-weight: bold;
    box-sizing: border-box;
  }
  
  nav a:not(.navbar-brand):hover {
      text-decoration: none;
     color: whitesmoke;
    font-weight: bold;
  }
  