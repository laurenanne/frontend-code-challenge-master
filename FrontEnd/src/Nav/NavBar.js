import React from "react";
import { NavLink } from "react-router-dom";
import { Navbar, Nav, NavItem } from "reactstrap";
import "./NavBar.css";

function NavBar() {
  return (
    <div>
      <Navbar className="navbar">
        <Nav>
          <NavItem>
            <NavLink to="/">Home</NavLink>
          </NavItem>

          <NavItem>
            <NavLink to="/signup">Sign Up</NavLink>
          </NavItem>
          <NavItem>
            <NavLink to="/admin">Users</NavLink>
          </NavItem>
        </Nav>
      </Navbar>
    </div>
  );
}

export default NavBar;
