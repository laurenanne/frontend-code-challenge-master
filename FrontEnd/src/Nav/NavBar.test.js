import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import NavBar from "./NavBar";

it("renders without crashing", function() {
  render(
    <MemoryRouter>
      <NavBar />
    </MemoryRouter>
  );
});

it("matches snapshot", function() {
  const { asFragment } = render(
    <MemoryRouter>
      <NavBar />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
