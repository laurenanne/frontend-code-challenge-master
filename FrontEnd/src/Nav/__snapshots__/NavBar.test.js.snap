// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <nav
      class="navbar navbar"
    >
      <div
        class="container-fluid"
      >
        <ul
          class="nav"
        >
          <li
            class="nav-item"
          >
            <a
              aria-current="page"
              class="active"
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class=""
              href="/signup"
            >
              Sign Up
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class=""
              href="/admin"
            >
              Users
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
</DocumentFragment>
`;
